### ISSUE TITLE
Window flicker and double titlebar

### ISSUE THREAD

**Issue by MioBlanco** [weisJ/darklaf#95]
Hi,
If a component that is very close to the edge of the frame tries to open a tooltip the window will start to flicker and once you maximize the window you will have two titlebars.

With this code you should be able to reproduce the issue:

````java   
public class Main
{
    public static void main( String[] args )
    {
        LafManager.install();
        LafManager.install( new DarculaTheme() );
        JPanel mainpanel = new JPanel( new BorderLayout() );
        JFrame frame = new JFrame();
        JTabbedPane pane = new JTabbedPane();
        JPanel panel1 = new JPanel();
        JLabel label = new JLabel( "Hover here" );
        label.setFont( label.getFont().deriveFont( 25f ) );
        label.setToolTipText( "This ToolTipText will cause the window to flicker and have 2 Titlebars" );
        pane.addTab( "Test1", panel1 );
        mainpanel.add( pane, BorderLayout.CENTER );
        mainpanel.add( label, BorderLayout.SOUTH );

        frame.add( mainpanel );
        Dimension DimMax = Toolkit.getDefaultToolkit().getScreenSize();
        frame.setMaximumSize( DimMax );

        frame.setExtendedState( JFrame.MAXIMIZED_BOTH );
        frame.setLocationRelativeTo( null );
        frame.setVisible( true );
        frame.setDefaultCloseOperation( JFrame.EXIT_ON_CLOSE );
    }
}
````

**Comment by weisJ** [comment]
Should be resolved now.

**Comment by Bios-Marcel** [comment]
May I recommend to link the commits that solve a problem in the respective issue? It's useful for future inspection of past issues and the likes :)

**Comment by weisJ** [comment]
> May I recommend to link the commits that solve a problem in the respective issue?

I'll add them in the future :) thank you for the suggestion. (btw this was fixed in 1d54edba05ce1e7d9a1b282cbede614416c8b38e)

>Github also has a feature where you can append "Fixes #IssueNumber" at the end of a commit message in order to automatically close issues.

Good to know. Didn't know about that :D
